docs/uds-packages/requirements/uds-package-requirements.md

```diff
@@ -69,7 +69,7 @@ Bronze packages:
 - [ ] **Must** deploy and operate successfully with Istio injection enabled in the namespace.
 - [ ] **Must** implement Journey testing, covering the basic user flows and features of the application (see [Testing Guidelines](https://github.com/defenseunicorns/uds-common/blob/main/docs/uds-packages/guidelines/testing-guidelines.md))
 - [ ] **Must** implement Upgrade Testing to ensure that the current development package works when deployed over the previously released one. (see [Testing Guidelines](https://github.com/defenseunicorns/uds-common/blob/main/docs/uds-packages/guidelines/testing-guidelines.md))
-- [ ] **Must** be capable of operating within an internet-disconnected (air-gapped) environment
+- [ ] **Must** be capable of operating within an internet-disconnected (airgap) environment
```

Comment:
```suggestion
- [ ] **Must** be capable of operating within an internet-disconnected (airgapped) environment
```

Reply:
Designers suggested  "Must be capabile of operating in an airgap (internet-disconnected) environment."